src/main/java/uk/gov/hmcts/ccd/domain/service/search/elasticsearch/ElasticsearchCaseSearchRequestFactory.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.hmcts.ccd.domain.service.search.elasticsearch;
+
+import io.searchbox.core.Search;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.ccd.ApplicationParams;
+
+@Component
+public class ElasticsearchCaseSearchRequestFactory extends CaseSearchRequestFactory<Search> {
+
+    @Autowired
+    public ElasticsearchCaseSearchRequestFactory(ApplicationParams applicationParams, CaseSearchQuerySecurity caseSearchQuerySecurity) {
+        super(applicationParams, caseSearchQuerySecurity);
+    }
+
+    @Override
+    protected Search createSearchRequest(String caseTypeId, String query) {
```

Comment:
any reason why this method has been declared as abstract and implemented in a subclass?

Reply:
Removed this class